### PR TITLE
Pin create-react-context due to non open source licence

### PR DESCRIPTION
**Pull request by Everspace**
See: https://github.com/react-navigation/react-navigation/issues/4934 for general gist and: https://github.com/jamiebuilds/create-react-context/commit/392ef2b40a4a6239ff63131c03797b7607708040 for the breaking change in a minor version.

Jamiebuilds is acting in bad faith to the open source community and this should not propagate down the line.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "printWidth": 80
   },
   "dependencies": {
-    "create-react-context": "^0.2.1",
+    "create-react-context": "0.2.1",
```
Comment (realityking) [inline package.json]: Should pin to 0.2.2 as the last MIT licensed version.

**Comment by crudh** [comment]
The license change seems to be reverted so I think this can be closed.

**Comment by bvancea** [comment]
Actually, the changes were reverted, but the revert was only included in 0.3.0. When installing create-react-context@0.2.3 the LICENSE still contains the following lines
```
Copyright (c) Jamie Kyle

This license is granted to everyone except for the following entities and
any of their subsidiaries:

- "Microsoft Corporation" (for working with ICE)
- "Palantir Technologies" (for working with ICE)
- "Amazon.com, Inc." (for abusive treatment of workers and for working with ICE)
- "Northeastern University" (for working with ICE)
- "Ernst & Young" (for working with ICE)
- "Thomson Reuters" (for working with ICE)
- "Motorola Solutions" (for working with ICE)
- "Deloitte Consulting LLP" (for working with ICE)
- "John Hopkins University" (for working with ICE)
- "Dell Inc" (for working with ICE)
- "Xerox Corporation" (for working with ICE)
- "Canon Inc" (for working with ICE)
- "Vermont State Colleges" (for working with ICE)
- "Charter Communications"/"Spectrum"/"Time Warner Cable" (for working with ICE)
- "LinkedIn Corporation" (for working with ICE)
- "United Parcel Service Co" (for working with ICE)
- "Walmart Inc" (for abusive treatment of workers)
- "Sears Holding Corporation" (for abusive treatment of workers)
- "Apple Inc" (for abusive treatment of workers)
- "Tyson Foods Inc" (for abusive treatment of workers)
- "Target Corporation" (for union busting and anti-union propaganda)
- "The H&M group" (for abusive treatment of workers)
- "Tesla, Inc" (for abusive treatment of workers)
```

The issue remains, and users associated with the above companies will not be able to use `@reach/router`, or libraries depending on `@reach/router`, like storybook.

**Comment by bvancea** [comment]
@ryanflorence Thank your for merging the PR. Is there any chance you can release a 1.2.2 version on npm?

**Pull request merged**